simplq/src/components/common/Button/Button.stories.jsx

```diff
@@ -16,6 +16,10 @@ export const JoinQButton = (props) => {
   return <StandardButton onClick={props.onClick}>Join Queue</StandardButton>;
```

Comment:
we could have changed the text here? Would you like to do a refactoring task?

Can you change "Q" in all these button names to "Queue"?